### PR TITLE
test: add image conversion properties for improved implementation

### PR DESCRIPTION
**Pull request by stijnmoreels**
Adding image conversion properties for more stronger testing with slightly improved/readable implementation.

Relates to #108

### CODE REVIEW

**Comment by stijnmoreels** [comment]
Dipping my toes into some Python testing. 😄

File: tests/test_images_conversion.py [hunk 1]
```diff
@@ -5,25 +5,66 @@
...
 import logging
-import numpy as np 
+import numpy as np
+from hypothesis import given, strategies as st
 
 image_path = 'tests/resources/images/lungs'
 
+def is_black_or_white(cell):
+    return cell == 0 or cell == 1
+def all_black_and_white(arr):
```
Comment (stijnmoreels) [inline tests/test_images_conversion.py:15]: Some help, I wanted to use `np.all` but seems only to work with 2D arrays.

Comment (stijnmoreels) [reply tests/test_images_conversion.py:15]: Aha, I think I can fix this with `.all()` which works also for multi-dimensional arrays.

**Review by ALaks96 — APPROVED** [review]

File: tests/test_images_conversion.py (hunk 1 above)
Comment (stijnmoreels) [reply tests/test_images_conversion.py:15]: I wanted to verify if the result 3D-array only contains 1 or 0 values. But seems that the `np.all` can only work with static ones and 2D-arrays. So, I did the assertion myself, but please correct me if you have something better. 😉

**Pull request merged**